website/content/en/docs/deployment/connect-kubeflow-dashboard.md

```diff
@@ -5,7 +5,7 @@ weight = 70
 +++
 
 ## Port-forward (Manifest deployment)
-
+> Note: Make sure you are in the repository root directory. 
```

Comment:
nit: Make sure you are in the the root of the repository before running the following commands.  Running `pwd` command should output `<path/to/kubeflow-manifests>` directory

Reply:
fixed.